spider_cms.py

```diff
@@ -627,10 +626,45 @@ def main(args):
     fd.close()
     os.rename(tmpname, "checkpoint.json")
 
-    logging.warning("Total processing time: %.2f mins" % (
+    logging.warning("Total processing time for history: %.2f mins" % (
                     (time.time()-starttime)/60.))
 
 
+    # Now that we have the fresh history, process the queues themselves.
+    if not args.process_queue: return 0
```

Comment:
Can you break this apart into two lines (just for consistency's sake).

Reply:
Done